linux/scripts/upload-to-debian.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

While this is not necessary for Linux, it doesn't hurt, and I'd like to try and keep our bash script boilerplate consistent for ease of maintenance.